src/screenComponents/jumpControls.cpp

```diff
@@ -18,10 +19,10 @@ GuiJumpControls::GuiJumpControls(GuiContainer* owner, string id)
     charge_bar->setPosition(0, -50, ABottomLeft)->setSize(50, GuiElement::GuiSizeMax);
     charge_bar->hide();
     
-    label = new GuiKeyValueDisplay(this, id + "_LABEL", 0.5, "Distance", "10.0");
+    label = new GuiKeyValueDisplay(this, id + "_LABEL", 0.5, tr("Distance"), "10.0");
```

Comment:
Might be good to have a context with this distance? Not all distances might translate the same in all languages?

Reply:
You are right. I actually added the context in another part of the file, but missed that one.

Reply:
Done.